mysql/requirements.in

```diff
@@ -1 +1,2 @@
-pymysql==0.8.0
+cryptography==2.3.1
```

Comment:
Why do we need this lib?

Reply:
From version 8.0, mysql uses a different way to authenticate and cryptography is required by newer versions of pymysql.